roles/mirror/tasks/route53.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Create AWS config dir
+  ansible.builtin.file:
+    path: /root/.aws
+    state: directory
+    mode: "0700"
+
+- name: Write AWS config file
+  vars:
+    aws_access_key_id:
+      "{{ lookup('community.hashi_vault.hashi_vault', 'kv/data/infra/aws/rsync.repo.almalinux.org/route53:access_key',
+      token=lookup('env', 'VAULT_TOKEN'), url=secrets_url) }}"
+    aws_secret_access_key:
+      "{{ lookup('community.hashi_vault.hashi_vault', 'kv/data/infra/aws/rsync.repo.almalinux.org/route53:secret_key',
+      token=lookup('env', 'VAULT_TOKEN'), url=secrets_url) }}"
+  ansible.builtin.template:
+    src: aws_config.j2
```

Comment:
Same as above, usually ansible-lint complaints if there is no user/group defined for file operations like these.